### ISSUE TITLE
New version of 'rich' deps break molecule from run

### ISSUE THREAD

**Issue by pavelpicka** [ansible-community/molecule#3010]
# Issue Type

- Bug report

# Molecule and Ansible details

```
ansible --version && molecule --version
{2.9,2.10}-{3.6,3.7,3.8}
```

Molecule installation method (one of):

- pip

Ansible installation method (one of):

- pip

Detail any linters or test runners used:
local, GHA, flake8 & yamllint, ansible-lint used

# Desired Behavior

molecule test --scenario-name our-test
should run the test

# Actual Behaviour

fails, on CI with "InvocationError" locally just RC=1 without any message

# Found

Only change from working behaviour is dependency rich is updated from ```'9.4.0'``` to ```'9.5.0'```. 
Downgrade is working workaround.

**Comment by btravouillon** [comment]
I nailed it to commit https://github.com/willmcgugan/rich/commit/12b268d223d2f4eed77f5d88ef875be4dfdc2170.
[enrich](https://pypi.org/project/enrich/#history) 1.2.3 is not compatible with the change in rich.

**Comment by greg-hellings** [comment]
For my own purposes I've currently pinned 'rich<9.5.0' in my deps. The behavior is present in both 9.5.0 and 9.5.1

**Comment by jsf9k** [comment]
Seeing the exact same behavior with respect to molecule and rich >= 9.5.0.  I came here to create an issue but saw that @pavelpicka beat me to it.

**Comment by jsf9k** [comment]
I believe @ssbarnea has now fixed this in pycontribs/enrich#21.
